homeassistant/components/modbus/__init__.py

```diff
@@ -139,11 +137,11 @@ def number(value: Any) -> Union[int, float]:
         vol.Optional(CONF_PRECISION, default=1): cv.positive_int,
         vol.Optional(CONF_SCALE, default=1): vol.Coerce(float),
         vol.Optional(CONF_OFFSET, default=0): vol.Coerce(float),
-        vol.Optional(CONF_MAX_TEMP, default=35): cv.positive_int,
-        vol.Optional(CONF_MIN_TEMP, default=5): cv.positive_int,
+        vol.Optional(ATTR_MAX_TEMP, default=35): cv.positive_int,
```

Comment:
Config constants should start with `CONF_*`.

Reply:
I do agree but these are imported from HA core/internal:
from homeassistant.components.climate.const import ATTR_MAX_TEMP, ATTR_MIN_TEMP
in climate there are a number of ATTR_* used in SCHEMA.

But anyhow I have done ATTR_* -> CONF_*